docs/setup/administrators/setup-notifications.md

```diff
@@ -2,10 +2,56 @@
 
 As mentioned in [External Integrations](../../getting-started/architecture.md#external-integrations), Hypha offers notifications via e-mail and/or Slack.
 
-## Email
+## Notification providers setup
+
+Hypha sends messages via configured providers such as email or Slack.
+
+### Email
 
 For information on configuring email notifications for Hypha, see the [E-mail section](configuration.md#e-mail-settings) of the configuration reference.
 
-## Slack
+### Slack
 
 For information on configuring Slack notifications for Hypha, see the [Slack section](configuration.md/#slack-settings) of the configuration reference.
+
+## Project report reminders
```

Comment:
I didn't want this to appear as if it were alike to "Email, Slack, ..." so I moved those into their own section before adding this one.